src/CAN/can_motor.cpp

```diff
@@ -0,0 +1,44 @@
+#include "can_motor.h"
+#include "../world_interface/real_world_constants.h"
+#include "CAN.h"
+#include "CANMotor.h"
+#include "CANUtils.h"
+
+namespace robot {
+class can_motor: public base_motor {
+	public: 
+		can_motor(robot::types::motorid_t motor, bool hasPosSensor) {
+			motor_id = motor;
+			posSensor = hasPosSensor;
+		}
+
+		bool hasPositionSensor() const {
```

Comment:
This functionality is probably common across all motor implementations, so can you move the definition to `base_motor`?

Reply:
Done